packages/aggregator/src/__tests__/analytics/filterAnalytics.test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { filterAnalytics } from '../../analytics/filterAnalytics';
+
+const analytics = [{ value: 1 }, { value: 2 }, { value: 3 }];
+
+describe('filterAnalytics()', () => {
+  describe('empty filter', () => {
+    const testData = [
+      ['empty array', []],
+      ['undefined', [undefined]],
+      ['empty object', [{}]],
+    ];
+
+    it.each(testData)('%s', (_, [filter]) => {
+      expect(() => filterAnalytics(analytics, filter)).not.toThrow();
+    });
+  });
+
+  describe('filter type', () => {
```

Comment:
It's a bit subjective but I believe that the tests here are very readable now! 🎉